xtask/src/test.rs

```diff
@@ -81,8 +88,8 @@ pub fn run_tests(shell: Shell, mut args: Arguments) -> anyhow::Result<()> {
             "--retries",
             &retries,
         ])
+        .args(nocapture_flags)
         .args(args.finish())
-        .quiet()
```

Comment:
question: Why are we removing this flag?

Reply:
Carelessness. I'm sorry.